src/java/voldemort/store/readonly/swapper/AdminStoreSwapper.java

```diff
@@ -223,10 +224,14 @@ private String fetch(String storeDir) {
             } catch(ExecutionException e) {
                 fetchErrors = true;
                 if(e.getCause() instanceof QuotaExceededException) {
-                    numQuotaExceededException++;
+                    //numQuotaExceededException++;
+                    QuotaExceededExceptions.add(node.getId());
```

Comment:
Can you please remove the commented code ? If required, we can use git history to recover the code. Having commented code is a bad smell.